src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -81,6 +83,20 @@ public ExportConfig(Map<String,Object> config) {
         this.streamStatements = toBoolean(config.get("streamStatements")) || toBoolean(config.get("stream"));
         this.writeNodeProperties = toBoolean(config.get("writeNodeProperties"));
         exportQuotes(config);
+        this.optimizations = (Map<String, Object>) config.getOrDefault("useOptimizations", Collections.emptyMap());
+        this.optimizationType = OptimizationType.valueOf(optimizations.getOrDefault("type", OptimizationType.UNWIND_BATCH.toString()).toString().toUpperCase());
+        validate();
+        if (optimizationType.equals(OptimizationType.UNWIND_BATCH_PARAMS)) {
+            this.batchSize = getUnwindBatchSize(); // batchSize is not considered because the data gets committed every UNWIND operation
```

Comment:
that is not true
data should be getting committed every batch size, which can be one or more unwind operations

Reply:
This is because of cypher-shell's problem of running `:param v=>2` inside of an open tx